docker-compose/Makefile

```diff
@@ -0,0 +1,152 @@
+PROJECT_HOME ?= ./openwhisk-master/
+WSK_CLI ?= $(PROJECT_HOME)/bin/wsk
+
+OPEN_WHISK_DB_PROTOCOL ?= http
+OPEN_WHISK_DB_HOST ?= localhost
+OPEN_WHISK_DB_PORT ?= 5984
+OPEN_WHISK_DB_PROVIDER ?= CouchDB
+OPEN_WHISK_DB_USERNAME ?= whisk_admin
+OPEN_WHISK_DB_PASSWORD ?= some_passw0rd
+DB_SUBJECTS_DBS ?= subjects
+OPEN_WHISK_DB_ACTIONS ?= whisk_actions
+OPEN_WHISK_DB_GW ?= whisk_gateway
+
+# Quick-Start is a simple way to get started with OpenWhisk locally
+#   1. at start it builds the project and the docker containers
+#   2. then it starts all components using docker-compose
+#   3. it runs a sample hello-world function
+#   To stop and cleanup the environment use: make destroy
+quick-start: download docker run quick-start-pause hello-world quick-start-info
+
+.PHONY: download
+download:
+	rm -rf ./openwhisk-master*
+	curl -O ./openwhisk-master.tar.gz -L https://api.github.com/repos/openwhisk/openwhisk/tarball/master > ./openwhisk-master.tar.gz
+	mkdir openwhisk-master
+	tar -xvf ./openwhisk-master.tar.gz --strip 1 -C openwhisk-master
+
+.PHONY: quick-start-pause
+quick-start-pause:
+	echo "waiting for the Whisk invoker to come up ... "
+	until $$(curl --output /dev/null --silent --head --fail http://localhost:8085/ping); do printf '.'; sleep 5; done
+	sleep 30
```

Comment:
good q. it's a simple workaround to wait for kafka to be ready but it could be done differently. `30` doesn't guarantee that it's enough or not.